test/events/failIssueRes.json

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
+1000 for this suggestion! In general, narrowing tests down to the smallest everything possible is a good thing (need to prove that the thing does multiples? Two is probably all you need. Not five. Want to test negative numbers? -1 is great)